### ISSUE TITLE
Italics in messages do not render correctly

### ISSUE THREAD

**Issue by LurkerHub** [EionRobb/purple-discord#429]
I noticed that some of my buddies' messages are not formatted correctly by Pidgin.
While they correctly show as italics (_text_) in the desktop client, they are shown in Pidgin as `*text *` or as `_text_`.

Example:

Pidgin:
![image](https://user-images.githubusercontent.com/20542915/201458661-95db7b2b-3c5d-43e8-befa-046aab04b3fa.png)

Native client:
![image](https://user-images.githubusercontent.com/20542915/201458693-2b1c60e9-0048-426e-8abe-47e41f975582.png)

To summarize the two bugs:
- Underscores do not translate to italics
- Asterisks translate to italics, unless there are trailing spaces
